X10D/src/Core/IntrinsicUtility.cs

```diff
@@ -0,0 +1,222 @@
+﻿#if NETCOREAPP3_0_OR_GREATER
+
+using System.Diagnostics.CodeAnalysis;
+using System.Diagnostics.Contracts;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
+
+namespace X10D.Core;
+
+/// <summary>
+///     Provides utility methods for SIMD vector that is currently missing on common hardware instruction set.
+/// </summary>
+public static class IntrinsicUtility
+{
+    /// <summary>
+    ///     <br>Multiply packed 64-bit unsigned integer elements in a and b and truncate the results to 64-bit integer.</br>
+    ///     <br>Operation:</br>
+    ///     <code>
+    ///     dest[0] = lhs[0] * rhs[0];
+    ///     dest[1] = lhs[1] * rhs[1];
+    ///     </code>
+    /// </summary>
+    /// <param name="lhs">Left vector.</param>
+    /// <param name="rhs">Right vector.</param>
+    /// <returns></returns>
```

Comment:
The `returns` element on the xmldoc is empty. Also `PlatformNotSupportedException` is not documented